ambry-cloud/src/main/java/com/github/ambry/cloud/azure/CosmosDataAccessor.java

```diff
@@ -70,10 +70,13 @@
   private static final String START_TIME_PARAM = "@startTime";
   private static final String END_TIME_PARAM = "@endTime";
   private static final String LIMIT_PARAM = "@limit";
+  private static final String ACCOUNT_ID_PARAM = "@accountId";
+  private static final String CONTAINER_ID_PARAM = "@containerId";
   private static final String EXPIRED_BLOBS_QUERY = constructDeadBlobsQuery(CloudBlobMetadata.FIELD_EXPIRATION_TIME);
   private static final String DELETED_BLOBS_QUERY = constructDeadBlobsQuery(CloudBlobMetadata.FIELD_DELETION_TIME);
   private static final String CONTAINER_BLOBS_QUERY =
-      "SELECT TOP %d * FROM c WHERE c.accountId=%d and c.containerId=%d";
+      "SELECT TOP %d " + LIMIT_PARAM + " FROM c WHERE c.accountId=" + ACCOUNT_ID_PARAM + " and c.containerId="
```

Comment:
I think Cosmos will reject this.  "%d" needs to be removed, and you need the "*" before FROM.